.travis.yml

```diff
@@ -1,7 +1,5 @@
 language: php
 
-dist: trusty
```

Comment:
I'm struggling with javascript scenarios on some newer version of Travis ubuntu for some time already (see [this](https://github.com/Zales0123/Sylius/commit/1500de3d03143db3d7b57da914d953fdaa79581a)). I don't know why it does not work properly. We don't see it in the build, but the problem is `WebDriver\Exception\InvalidCookieDomain: invalid cookie domain`. Maybe you've already encountered it?

Nevertheless, for this PR, can't we stay with Ubuntu Trusty yet? Or PHP 7.4 is not available on it?